src/main.cpp

```diff
@@ -234,10 +250,16 @@ int wmain(int argc, wchar_t* argv[])
             subCommand = CheckStartSubCommands(argv[currentArg]);
         }
 
+        if (noAdminSpecified && !noCpuSamplingSpecified) {
+            std::wcout << L"ERROR: you cannot specify both no admin privileges (with the [/noadmin] flag) and have CPU sampling on (without the [/nocpusampling] flag). CPU sampling requires admin privileges." << std::endl;
+            std::wcout << L"vcperf.exe /start [/noadmin] [/nocpusampling] [/level1 | /level2 | /level3] sessionName" << std::endl;
+            return E_FAIL;
+        }
```

Comment:
I think this should be removed so that users dont **have** to specify `/nocpusampling` when using `/noadmin`. `/noadmin` implies no CPU samples so it seems like forcing users to include the second switch might be seen as annoying.

Reply:
I removed this section and instead, when /noadmin is provided I also set noCpuSamplingSpecified to true.